OasisDEX/oasis-borrow
# Av1 and av2 common library usage

# [Av1 and av2 common library usage](https://app.shortcut.com/oazo-apps/story/7589/av1-av2-common-library-usage)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- replaced `decodeTriggerData` with `decodeTriggerDataAsJson` from common library
  
## How to test 🧪
  <Please explain how to test your changes>

- try to add, remove and update maker and aave triggers (correct numbers should be displayed)
